composer/workflows/airflow_db_cleanup.py

```diff
@@ -319,37 +315,27 @@ def build_query(
     airflow_db_model,
     age_check_column,
     max_date,
-    keep_last,
-    keep_last_filters=None,
-    keep_last_group_by=None,
+    dag_id=None
 ):
     query = session.query(airflow_db_model)
 
     logging.info("INITIAL QUERY : " + str(query))
 
-    if not keep_last:
-        query = query.filter(
-            age_check_column <= max_date,
-        )
+    if airflow_db_model == DagRun:
+        newest_dagrun = session.query(airflow_db_model) \
```

Comment:
Can you, please, avoid using "\" to wrap statement (here and below).

From pep8 guidelines:
"""
The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.
"""

Reply:
You meant backslashes, right? Probably formatting hidden that. I agree with you, thank you for noticing that.